### PR TITLE
Mega dropdown menu links update

### LINKED ISSUE #1812: Make sure all the links on the mega menu dropdown points to the internal beta
The [links](https://github.com/Qiskit/qiskit.org/blob/feature/textbook-demo/constants/megaMenuLinks.ts) in the [AppMegaDropdownMenu](https://github.com/Qiskit/qiskit.org/blob/feature/textbook-demo/components/ui/AppMegaDropdownMenu.vue) component should point to the new internal beta server.

---
User story: #1788 
Related: #1808, #1803, #1807

### PR DESCRIPTION
**Pull request by korgan00**
Fix #1812 

Links updated to point to the internal beta.

### CODE REVIEW

**Review by eddybrando — APPROVED** [review]
LGTM! Thanks 🙂

File: constants/megaMenuLinks.ts [hunk 1]
```diff
@@ -451,81 +473,156 @@ const INVESTIGATING_Q_HW_USING_Q_CIRCUITS : MegaDropdownMenuGroup = {
...
+        action: `${actionPrefix} > ${sectionLearningQml} > variational-classification`
+      }
+    }
+  ]
+}
+
 type MegaDropdownMenuColumn = MegaDropdownMenuGroup[]
 const COLUMN_1: MegaDropdownMenuColumn = [
   PREREQUISITES,
+  LEARNING_QML,
```
Comment (y4izus) [inline constants/megaMenuLinks.ts:610]: Why did you add the QML content on the first column?

Comment (y4izus) [reply constants/megaMenuLinks.ts:610]: I'm going to ask to the product team :)

File: constants/megaMenuLinks.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ const PREREQUISITES: MegaDropdownMenuGroup = {
     label: 'Prerequisites',
     url: `${baseUrl}${pathPrerequisites}`,
     segment: {
-      action: `${actionPrefix} > ${sectionPrerequisites}`
+      action: `${actionPrefix} > ${sectionPrerequisites} > whole-section`
```
Comment (y4izus) [inline constants/megaMenuLinks.ts:35]: You can create a variable for `whole-section` because you use it several times

**Review by y4izus — CHANGES_REQUESTED** [review]
Thank Goyo! Just one more comment :)

**Review by y4izus — APPROVED** [review]

**Pull request merged**